views/includes/inputs/fields/is-still-a-beneficial-owner-input.html

```diff
@@ -2,14 +2,20 @@
   {% include "includes/inputs/date/ceased-date-input.html" %}
 {% endset -%}
 
+{% if beneficialOwnerPronoun == "they" %}
+  {% set legendText = "Are they still a beneficial owner for the overseas entity?" %}
```

Comment:
stillBoText? Usually named to do with the question

Reply:
good shout, fixed :D